README.md

```diff
@@ -135,9 +135,13 @@ spec:
   image: registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.9
   provider: BAREMETAL
   ssh:
+    ## ssh的登录密码，如果使用的密钥登录则无需设置
```

Comment:
This is English document, may be you could using English? 
Chinese docs are in the docs/README_zh.md

Reply:
I have changed the configuration as required